desci-server/Dockerfile.ci

```
@@ -0,0 +1,29 @@
+FROM node:20.18.1-bullseye-slim
+
+RUN apt-get -qy update && apt-get -qy install openssl curl socat jq dumb-init
+
+RUN mkdir /app && chown -R node:node /app
+
+WORKDIR /app
+
+COPY --chown=node:node ./package.json ./yarn.lock .
```

Hadolint (2.14.0)

[error] COPY with more than 2 arguments requires the last argument to end with /

(DL3021)